src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import os
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def user_filesystem(tmp_path):
+    directory = Path(tmp_path)
+    os.chdir(directory)
+
+    existing_dir = Path(tmp_path).resolve() / "existing_dir"
+    existing_dir.mkdir(parents=True, exist_ok=True)
+
+    existing_file = Path(tmp_path).resolve() / "existing_file.py"
```

Comment:
We need the following files:
```
directory/good_data.chi   # a valid chi file
directory/good_data.xy   # valid data in xy format
directory/unreadable_file.txt  # a text file that will raise an error
directory/binary.pckl             # some kind of unreadable binary file, maybe some kind of pickle file or sthg
```
then all these same files written in another directory called `input_directory`

To build this we could have the files saved in some directory in tests and then copy them to `tmp_dir` as part of the fixture, or we could write the by saving the contents in a py file somewhere and doing writes.  The data only needs to be four lines long or so, but valid format so that loadData will read it correctly.